docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -186,10 +186,10 @@ tasks.named('compileKotlin', KotlinCompilationTask) {
 
 ### Attributes common to JVM and JS
 
-| Name | Description | Possible values |Default value |
-|------|-------------|-----------------|--------------|
-| `apiVersion` | Restrict the use of declarations to those from the specified version of bundled libraries | "1.3" (DEPRECATED), "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" |  |
-| `languageVersion` | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" |  |
+| Name | Description | Possible values                                                                          |Default value |
```

Comment:
I suppose that these spaces are automatically added by IDEA, not you; but anyway, it's better not to change tables this way because on laptops with not very wide screens, such tables are hard to read (need to scroll horizontally):

<img width="777" alt="Screenshot 2023-02-09 at 15 02 46" src="https://user-images.githubusercontent.com/78360457/217834575-d037e187-74b5-42aa-bc14-86b5edee8021.png">

We have a "limiter" setting in IDE; by default, it's 120 symbols. Please, avoid making lines longer than this value (the limiter visibly helps you).

I've made changes to return formatting back because I anyway opened IDE to check line length. I'll add my commit soon.